packages/debugger/src/config.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-import { murmur2 } from './hash';
+import { murmur2 } from '@jupyterlab/apputils';
```

Comment:
Linking to https://github.com/jupyterlab/debugger/pull/549#issuecomment-706299440 where it was discussed whether `murmur2` should be exposed as part of the JupyterLab API.

Maybe we should consider keeping it "private". Would it possible to directly use a field from the notification as id instead, for example `createdAt`?

Reply:
I switched to build the id in the backend (useful for testing too).

Reply:
:+1: